templates/javascript-ethers/hardhat.config.js

```diff
@@ -9,6 +9,7 @@ const network = NETWORK
       [NETWORK]: {
         ...(PROVIDER_URL && { url: PROVIDER_URL }),
         ...(MNEMONIC && {
+          // TODO: Should this be "accounts" (plural)?
```

Comment:
I'm confused about this comment 😁

Reply:
I found it confusing that you specify a mnemonic to accounts, anyway I'll resolve this in next (final) PR. You can ignore thuis.